rules/core/declaration-block-no-shorthand-property-overrides /correct.css

```diff
@@ -0,0 +1,20 @@
+a {
+    padding: 10px;
+    padding-left: 20px;
+}
+
+
```

Comment:
Lot of spacing

Reply:
yeah was trying to show that the one in the middle go together as a single example